ZenPacks/zenoss/ZenPackLib/tests/ZPLTestBase.py

```diff
@@ -30,6 +30,12 @@
 from ZenPacks.zenoss.ZenPackLib.lib.helpers.loaders import WarningLoader
 from ZenPacks.zenoss.ZenPackLib.lib.helpers.ZenPackLibLog import ZPLOG
 
+run_tests = False
```

Comment:
Globals should be uppercase: `RUN_TESTS`.

Also I think the name and the name of its corresponding environment variable are a bit of a misnomer. It would make me think that it enables/disables all ZPL tests. Maybe broken is a harsh word, but something like `RUN_BROKEN_TESTS` or `RUN_FLAKY_TESTS` would convey the meaning better.